build/dev/startup/grpc_backend.sh

```diff
@@ -16,7 +16,7 @@ if [[ $DEBUG_ON == 1 ]]; then
   -reflect_api \
   -log_format console \
   -dump_requests \
-  -accepted_jwt_audiences localhost,host.docker.internal \
+  -accepted_jwt_audiences localhost,,host.docker.internal,local-gateway,dss_sandbox_local-dss-http-gateway_1 \
```

Comment:
nit: I don't think we want the extra comma here as that will allow an empty audience (other additions are good)